packages/react/src/components/Tag/Tag-story.js

```diff
@@ -36,6 +49,15 @@ const props = {
       onClose: action('onClose'),
     };
   },
+  icon() {
+    let iconToUse = iconMap[select('Icon (icon)', icons, 'Tag16')];
+    return {
+      ...this.regular(),
+      onClick: action('onClick'),
+      onClose: action('onClose'),
+      renderIcon: iconToUse,
+    };
```

Comment:
I think we can just inline this since we aren't checking for falsiness or the svgData property

```suggestion
    return {
      ...this.regular(),
      onClick: action('onClick'),
      onClose: action('onClose'),
      renderIcon: iconMap[select('Icon (icon)', icons, 'Tag16')],
    };
```